packages/external-integration/src/external-payment-strategy.ts

```diff
@@ -32,29 +31,23 @@ export default class ExternalPaymentStrategy implements PaymentStrategy {
         try {
             await this._paymentIntegrationService.submitPayment({ ...payment, paymentData });
         } catch (error) {
-            if (isRequestError(error)) {
```

Comment:
Do we know if this piece of code was being used by `laybuy`?

Reply:
I just changed the condition to line 35.